README.md

```diff
@@ -39,6 +40,31 @@ Important notes:
     make tests_cpp_region
     make tests_unit
 
+#### Install nupic.bindings for nupic:
+    cd $NUPIC_CORE
+    export NUPIC_CORE_RELEASE=$NUPIC_CORE/build/release
+    python setup.py install
+
+> **Note**: set `NUPIC_CORE_RELEASE` to the location where `nupic.core` was installed.
+
+If you get a gcc exit code 1, you may consider running this instead:
+
+     [sudo] CC=clang CXX=clang++ python setup.py install --user
+
+If you are installing on Mac OS X, you should add the instruction `ARCHFLAGS="-arch x86_64"` before the python call:
```

Comment:
"If you are installing on Mac OS X, you _must_ add the instruction `ARCHFLAGS="-arch x86_64"` before the python call"